### ISSUE TITLE
[prometheus-node-exporter] Question on DaemonSets for node exporter

### ISSUE THREAD

**Issue by dbbc96** [prometheus-community/helm-charts#3862]
So pardon for this generic question but thought i would ask it here in hopes that someone might know.  I just was starting testing out Prometheus.  I deployed the kube-prometheus-stack chart if it matters or not.  

 My question is why is node exporter needing a large number of pods running as a DaemonSets.  Basically what i am seeing is that i have multiple pods managed by a DS that can't be scheduled since i have only 1 node.  When i look at the DS for it, its starting out with 5 and a recommended value.

**Comment by dbbc96** [comment]
just curious if anyone has any input on this.

**Comment by dbbc96** [comment]
anyone have thoughts on this?

**Comment by kfox1111** [comment]
DaemonSets run one pod per node. Node Exporter needs to run one per node to gather node metrics.

**Comment by dbbc96** [comment]
yeah i get that however when i had tested this, i had 3 nodes and so it had running exporters on those however there were ones that still didn't get scheduled.  my guess is that there might be different node exporters that need to run on each node.  so there wouldn't be just 1 exporter on each node. but i am guessing on this.

**Comment by ShadowJonathan** [comment]
You might have node taints that affect their scheduling, double-check if you've set the right tolerations

**Comment by dbbc96** [comment]
@ShadowJonathan I will check and let you know.  I don't think I did.
